SampleApplications/SDK/Opc.Ua.Server/Server/ServerInternalData.cs

```diff
@@ -644,6 +644,10 @@ private void CreateServerObject()
                     m_defaultSystemContext,
                     m_configuration.ServerConfiguration.DiagnosticsEnabled);
 
+                ConfigurationNodeManager configurationNodeManager = m_diagnosticsNodeManager as ConfigurationNodeManager;
```

Comment:
would lead to some confusion for people who the codebase...

---
In reply to: [155814557](https://github.com/OPCFoundation/UA-.NETStandard/pull/284#discussion_r155814557) [](ancestors = 155814557)